api/v1/installation_types.go

```diff
@@ -463,6 +463,10 @@ type CalicoNetworkSpec struct {
 	// +optional
 	// +kubebuilder:validation:Enum=Enabled;Disabled
 	ContainerIPForwarding *ContainerIPForwardingType `json:"containerIPForwarding,omitempty"`
+
+	// SysctlTuning configures sysctl parameters for tuning plugin
+	// +optional
+	SysctlTuning *map[string]string `json:"sysctlTuning,omitempty"`
```

Comment:
After chatting with some other folks it would be better to replace this with:
```
Sysctl []Sysctl `json:"sysctl,omitempty"`
```
and add
```
type Sysctl struct {             
    // +kubebuilder:validation:Enum=net.ipv4.tcp_keepalive_intvl;net.ipv4.tcp_keepalive_probes;net.ipv4.tcp_keepalive_time
    Key   string `json:"key"`  
    Value string `json:"value"`
} 
```

This also provides apply time validation rather than operator load time validation. In other words it will block `kubectl apply` if there is an incorrect value instead of waiting for the error to be found when the operator processes the field.

Reply:
thanks!

Reply:
Sorry for not being clear but part of this suggestion was to stop using a map as that is not K8s idomatic (except in annotations and labels).
With my suggestion the config would look like
```
sysctl:
  - key: net.ipv4.tcp_keepalive_intvl
    value: "15"
  - key: net.ipv4.tcp_keepalive_probes
    value: 6
```

Reply:
got it, changed as requested